cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -52,6 +52,7 @@
     "file": "document",
     "line": "line",
     "paint": "nonWhitespaceSequence",
+    "small paint": "nonWhitespaceOrQuoteSequence",
```

Comment:
"nonWhitespaceOrQuoteSequence" is not an optimal name. "nonWhitespaceOrPairSequence" maybe?
@pokey Any ideas?

Reply:
Right, not accurate any longer. Maybe "nonWhitespaceSurroundingPairIntersect"? Pretty long but conceptually accurate.

Reply:
We generally prefer less verbose names for the identifier, to give us more flexibility to tweak behaviour in the future.  Also wrecks layout in the cheatsheet if they're too long 😅

Maybe `boundedNonWhitespaceSequence`?

Reply:
I personally quite like that suggestion.